threenums.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+int main ()  {
```

Comment:
No comments and you didn't properly handle the case where two numbers are equal and also the greatest number (so, say, 9 9 1). To avoid mishandling edge cases, test your code thoroughly!